pools.json

```diff
@@ -188,5 +188,12 @@
         "searchStrings": [
             "t1cVYU7yj2doV33B1Acn586FXn2GFSuwi3f"
         ]
+    },
+    {
+    	"poolName": "Speedpool",
+    	"url": "https://equihash.speedpool.top/getting_started",
+    	"searchStrings": [
+      	    "equihash.speedpool.top"
```

Comment:
The search string has to be your zelcash pool address. Address to which your pool is mining to.

Reply:
yes it is, to mine zelcash the url = stratum+tcp://equihash.speedpool.top: 8043
The searchstring parameter is = url = equihash.speedpool.top

Reply:
Search string parameter has to be your zelcash transparent address - starts with t1, look at many examples above your addition. Insight explorer is looking for an address to which the coins were mined to.